lib/src/builder.dart

```diff
@@ -98,7 +99,11 @@ class GeneratorBuilder extends Builder {
 
 Stream<GeneratedOutput> _generate(LibraryElement unit,
     List<Generator> generators, BuildStep buildStep) async* {
-  for (var element in getElementsFromLibraryElement(unit)) {
+  var elements = safeIterate(getElementsFromLibraryElement(unit), (e, [_]) {
+    log.severe('Failed to resolve ${buildStep.inputId}.');
+    log.fine('Resolve error details:\n$e');
```

Comment:
It's no longer synchronously fatal.
This log does show up in bazel with --define=DART_CODEGEN_LOG_LEVE=fine

Would you prefer if I move it earlier than the log.severe for safety?